pkg/daemon/ceph/client/filesystem.go

```diff
@@ -326,3 +369,34 @@ func deleteFSPool(context *clusterd.Context, clusterInfo *ClusterInfo, poolNames
 	}
 	return DeletePool(context, clusterInfo, name)
 }
+
+// WaitForNoStandbys waits for all standbys go away
+func WaitForNoStandbys(context *clusterd.Context, clusterInfo *ClusterInfo, timeout time.Duration) error {
+	err := wait.Poll(3*time.Second, timeout, func() (bool, error) {
+		mdsDump, err := GetMDSDump(context, clusterInfo)
+		if err != nil {
+			logger.Errorf("failed to get fs dump, %v", err)
+			return false, nil
+		}
+		return len(mdsDump.Standbys) == 0, nil
+	})
+
+	if err != nil {
+		return errors.Wrapf(err, "timeout waiting for no standbys")
+	}
+	return nil
+}
+
+func GetMDSDump(context *clusterd.Context, clusterInfo *ClusterInfo) (*MDSDump, error) {
+	args := []string{"fs", "dump"}
+	cmd := NewCephCommand(context, clusterInfo, args)
+	buf, err := cmd.Run()
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to dump fs info")
+	}
+	var dump MDSDump
+	if err := json.Unmarshal(buf, &dump); err != nil {
+		return nil, errors.Wrapf(err, "failed to unmarshal fs dump")
```

Comment:
```suggestion
		return nil, errors.Wrapf(err, "failed to unmarshal fs dump. %s", buf)
```

Printing the output is helpful